bedrock/firefox/templates/firefox/whatsnew/fx64/whatsnew-fx64-base.html

```diff
@@ -62,8 +68,8 @@ <h1 class="c-page-header-up-to-date">{{ latest }}</h1>
    include_cta=True
   ) %}
   <p class="show-fxa-supported-signed-out show-fxa-default">
-    <a href="https://accounts.firefox.com/signup?service=sync&context=fx_desktop_v3&entrypoint=mozilla.org-fx.wnp64&utm_content=%2Ffirefox%2FWNP%2F64%2F&utm_campaign=wnp64&utm_source=wnp-64" class="mzp-c-button mzp-t-download">{{ button_create }}</a><br>
-    <span class="wn64-signin">{{ login_desc }} <a href="https://accounts.firefox.com/?action=email&service=sync&context=fx_desktop_v3&entrypoint=mozilla.org-fx.wnp64&utm_content=%2Ffirefox%2FWNP%2F64%2F&utm_campaign=wnp64&utm_source=wnp-64">{{ login_link }}</a></span>
+    <a {{ fxa_link_fragment(entrypoint='mozilla.org-fxa.wnp64', action='signup', utm_params=fxa_utm_params) }} class="mzp-c-button mzp-t-download">{{ button_create }}</a><br>
```

Comment:
The entrypoint here should be `mozilla.org-fx.wnp64` to match the rest (you have an extra `a` between `f` and `x`.

Reply:
Fixed for both links.